home/src/main/resources/config/example.runtime.properties

```diff
@@ -456,6 +456,10 @@ createAndLink.providers = doi, pmid
 #comma separated list of mime types allowed for upload
 #fileUpload.allowedMIMETypes = image/png, application/pdf
 
+# Feature toggle for forgot password functionality
```

Comment:
We probably want to provide the available options here. I assume `[enabled, disabled]`.